### PR TITLE
use 0.3.4-ai2 version

### PR DESCRIPTION
**Pull request by cristipp**
Publish Grobid artifacts to the Nexus resolver.

We published on top of https://github.com/allenai/grobid/pull/1, possibly these instructions are broken until we submit that PR.

### CODE REVIEW

**Comment by cristipp** [comment]
@schmmd @severian FYI

File: README.ai2.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+grobid-core/lib/fr/limsi/wapiti/wapiti/1.5.0/wapiti-1.5.0.pom
+```
+
+The mvn command is simply:
+
+```
+mvn deploy:deploy-file -Dfile=grobid-core/lib/$ORG_SLASH/$NAME/$VER/$NAME-$VER.jar -DpomFile=grobid-core/$ORG_SLASH/$NAME/$VER/$NAME-$VER.pom -DgroupId=$ORG_DOT -DartifactId=$NAME -Dversion=$VER -Dpackaging=jar -Durl=http://utility.allenai.org:8081/nexus/content/repositories/public-releases/ -DrepositoryId=ai2
+```
+
+TODO(Patrick): Automatize the process, perhaps just use standard resolvers instead of local package copies.
```
Comment (schmmd) [inline README.ai2.md:87]: TODO: publish to JCenter/BinTray.

**Comment by cristipp** [comment]
Ping!

**Comment by cristipp** [comment]
@rodneykinney Can we detail the MetaEval validation procedure?

**Comment by andrewlmurray** [comment]
@cristipp looks good to me

**Comment by cristipp** [comment]
Thanks. I'll submit now.

**Pull request merged**